src/Javascript/Control/Select2v4.php

```diff
@@ -40,12 +39,10 @@ class Select2v4
 
     protected function __construct(Session $session, NodeElement $base_select)
     {
-        if ( ! $session->getDriver() instanceof Selenium2Driver) {
-            throw new UnsupportedDriverActionException(
-                __CLASS__.' needs a selenium driver, got %s',
-                $session->getDriver()
-            );
-        }
+        //guard against drivers which don't support JS
```

Comment:
Maybe worth extracting this to a static method of it's own - `Ingenerator\BehatSupport\Mink\BrowserChecker::requireJavascript()` or something? 

We don't use it anywhere else here but could reuse it in the projects, then the intent is clear and we can rejig exactly how we check / what we accept in future if necessary?

Reply:
Yeah that's a good plan